### PR TITLE
Fix the API link in the "Resources" menu dropdown.

### LINKED ISSUE #218: [Bug] Fix nav link to the API
The navigation link in the "Resources" menu dropdown should link to https://api.openverse.engineering/v1/

CleanShot 2021-09-15 at 16 13 50@2x

### PR DESCRIPTION
**Pull request by theshakeabhi**
## Fixes

Fixes #218  by @zackkrida 

## Description
Initially the API on the Dropdown of Resources in the Navbar was being directed to  https://api.openverse.engineering, now I have redirected it to  https://api.openverse.engineering/v1/ as mention on #218 .

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/EmbeddedNavSection.vue [hunk 1]
```diff
@@ -104,7 +104,7 @@
             {{ $t('header.feedback-nav-item') }}
           </NuxtLink>
           <a
-            href="https://api.creativecommons.engineering/"
+            href="https://api.creativecommons.engineering/v1/"
```
Comment (zackkrida) [inline src/components/EmbeddedNavSection.vue:107]:
```suggestion
            href="https://api.openverse.engineering/v1/"
```

The main goal here was to switch from `creativecommons` to `openverse` 😄

File: src/components/NavSection.vue [hunk 2]
```diff
@@ -101,7 +101,7 @@
             {{ $t('header.feedback-nav-item') }}
           </NuxtLink>
           <a
-            href="https://api.creativecommons.engineering/"
+            href="https://api.creativecommons.engineering/v1/"
```
Comment (zackkrida) [inline src/components/NavSection.vue:104]:
```suggestion
            href="https://api.openverse.engineering/v1/"
```

**Review by zackkrida — CHANGES_REQUESTED** [review]
Thank you for adding the `/v1` to the url! There was one more edit needed, switching from `creativecommons` to `openverse`. You can commit my suggestions right in the GitHub user interface and I will review again after. Thank you, and welcome to open source 😸

**Review by sarayourfriend — APPROVED** [review]
Thanks for the contribution! And welcome to Openverse 😁 🚀

**Comment by theshakeabhi** [comment]
I got a bit too excited as it was first PR and made those silly mistakes. I will be more aware next time. Thank you @zackkrida for helping me with this issue.

**Review by zackkrida — APPROVED** [review]

**Pull request merged**